tests/playwright-tests/admin/pages/ManagePublicationReleasePage.ts

```diff
@@ -0,0 +1,18 @@
+import { Locator, Page } from '@playwright/test';
+
+export default class ManagePublicationReleasePage {
+  readonly page: Page;
+  readonly createNewRelease: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    // Locators
+    this.createNewRelease = page.locator(
+      '//a[contains(text(),"Create new release")]',
+    );
+  }
+
+  async createNewReleaseLink() {
```

Comment:
Would this be better named as an action, like "clickCreateNewReleaseLink"?  I think it would improve the readability of a test that is using this page object.